### PR TITLE
Seed Project Path Update

### PR DESCRIPTION
**Pull request by mehreencs87**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/native-platforms/cordova/01-login.md [hunk 1]
```diff
@@ -14,10 +14,11 @@ This tutorial and seed project have been tested with the following:
 :::
 
 <%= include('../../_includes/_package', {
-  pkgRepo: 'auth0-cordova',
```
Comment (mpaktiti) [inline articles/native-platforms/cordova/01-login.md:17]: @mehreencs87 Hi, another change was introduced and we need to apply it before this PR is merged. `pkgOrg` was added:
- if the sample is under `auth0-samples` repo add the line: `pkgOrg: 'auth0-samples'`
- if the sample is under `auth0` repo add the line: `pkgOrg: 'auth0'`
  Details: https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#seed-projects

Can you please update this? Thanks.

**Comment by mehreencs87** [comment]
@mpaktiti check the latest commit.

File: articles/native-platforms/cordova/01-login.md [hunk 2]
```diff
@@ -14,10 +14,12 @@ This tutorial and seed project have been tested with the following:
...
+  pkgOrg: 'auth0-samples',
+  pkgRepo: 'auth0-cordova-samples',
+  githubUrl: 'https://github.com/auth0-samples/auth0-cordova-samples/tree/master/00-Starter-Seed',
```
Comment (mpaktiti) [inline articles/native-platforms/cordova/01-login.md]: @mehreencs87 Can you please set `githubUrl` to `https://github.com/auth0-samples/auth0-cordova-samples/tree/master/00-Starter-Seed/basic-sample`? Thanks!

**Comment by mehreencs87** [comment]
@mpaktiti updated.

**Pull request merged**